nerdtree_plugin/fs_menu.vim

```diff
@@ -55,7 +55,14 @@ function! s:promptToDelBuffer(bufnum, msg)
         " Is not it better to close single tabs with this file only ?
         let s:originalTabNumber = tabpagenr()
         let s:originalWindowNumber = winnr()
-        exec "tabdo windo if winbufnr(0) == " . a:bufnum . " | exec ':enew! ' | endif"
+        let s:listedBufferCount = len(getbufinfo({'buflisted':1}))
```

Comment:
Note that `getbufinfo()` is not supported in Vim 7.4.

Reply:
I can't approve the change until this is fixed.

Reply:
Thanks for the find. I wasn't aware that getbufinfo is not supported in 7.4. Will work on that.

Reply:
Below works on vim 7.2 in my linux machine
```vim
:echo len(filter(range(1, bufnr('$')), 'buflisted(v:val)'))
```
I Need to find a clean way to do this

Reply:
I'm fine with it either way - old syntax or both.

Reply:
@mboughaba, I'll eventually push for a fork that will break with Vim 7 and only support Vim 8. No point in doing it right now though, adoption is not wide enough to to warrant the move.

Believe me, if we break compatibility with 7 at this time, complaints will flow!